projects/languages/java/controllers/Slave/Slave.java

```diff
@@ -64,12 +65,11 @@ public void run() {
 
     // perform a simulation steps and leave the loop when the simulation is ended
     while (step(timeStep) != -1) {
-
       // Read sensors, particularly the order of the supervisor
       if (receiver.getQueueLength()>0){
         String message = new String(receiver.getData());
         receiver.nextPacket();
-        System.out.println("I should "+message+"!");
+        System.out.println("I should "+AnsiCodes.RED_FOREGROUND+message+AnsiCodes.RESET+"!");
```

Comment:
Just wondering if it is normal to not add spaces between "+" signs

Reply:
No, I just kept the existing coding style for this file (which is not nice I agree), but it's out of the scope of this PR to fix that, as we should fix it in the whole file.